server/model/unused-deductions/unusedDeductionsMessageViewModel.ts

```diff
@@ -90,6 +91,10 @@ export default class UnusedDeductionsMessageViewModel {
       reviewMessage = 'review tagged bail to calculate'
     }
 
+    if (this.remandBannerVisible) {
```

Comment:
Can this be moved up to the switch statement? I'm not sure why its after the message has been partially constructed